.changeset/happy-garlics-admire.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This changeset should be removed since the update happened in another PR and that PR had a changeset

Reply:
✅ done, thanks